Changelog.md

```diff
@@ -36,8 +36,9 @@ and this project follows to [Ragnarök Versioning Convention](https://gist.githu
 
 ### Changed
 
-- Overhauled textures of AAC Honey Badger, Beowulf, HK 417, M16A1, 100 Rounds NATO Drum Magazine, revamped textured sight set and the pink camo skin
+- Overhauled textures of AAC Honey Badger, Beowulf, HK 417, M16A1, 100 Rounds NATO Drum Magazine and the pink camo skin
 - Overhauled animations of the M40A6
+- Overhouled the texture of the sights
```

Comment:
No, you actually need to list everything you overhaul it's the point of a changelog

Reply:
Changelog.dm is here